pkg/mock/mockcluster/mockcluster.go

```diff
@@ -659,3 +661,24 @@ func (mc *Cluster) CheckRegionUnderSuspect(id uint64) bool {
 func (mc *Cluster) ResetSuspectRegions() {
 	mc.suspectRegions = map[uint64]struct{}{}
 }
+
+func (mc *Cluster) GetTiDBs() []*core.TiDB {
+	return mc.tidbList
+}
+
+func (mc *Cluster) GetTiDB(address string) *core.TiDB {
+	for _, tidb := range mc.tidbList {
+		if tidb.Address == address {
+			return tidb
+		}
+	}
+	return nil
+}
+
+func (mc *Cluster) AddTiDB(tidbs []*core.TiDB) {
+	mc.tidbList = append(mc.tidbList, tidbs...)
```

Comment:
Should we check if some of the TiDB instances have been already added?

Reply:
good point